Mood sentiment analyzer

devpladinc opened the pull request:
# Description

Initial PR: Added a script that will check the sentiment or mood of a given text inside a text file. 

## Type of change
Initial PR
Please delete options that are not relevant.

- [x] New Script (non-breaking change which adds a new script to the repository)
- [x] This change requires a documentation update

## Has This Been Tested?

- [x] I confirm that I have tested my script locally and verified that it works as intended.

## Checklist:

- [x] I have gone through the contributing guidelines before making this Pull Request.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

rijusougata13 commented:
hi! whats those step1.png and step4.png in the image folder? If those are not necessary please remove those images.

devpladinc commented:
> hi! whats those step1.png and step4.png in the image folder? If those are not necessary please remove those images.

Fixed merge conflicts and removed guide images. Thanks!
